### PR TITLE
project-id-introduction

### PR DESCRIPTION
**Pull request by resmaa**
Introduction Page (ProjecUID)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/902#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/api/1-introduction.md [hunk 1]
```diff
@@ -93,7 +93,12 @@ The version information is part of the API URI like `v1alpha1`, `v1`. Future API
 
 # Scope
 
-Palette applications operate in *Tenant* and *Project* scopes. The resources can be logically grouped as projects and API requests within a project should carry the project uid in the context. The project scope can be specified in the API request as an HTTP header with the key as `projectUid` and value as the &lt;project uid&gt;
+Palette supports applications to operate under [**Tenant** and **Project**](/user-management/palette-rbac#accessmodes) scope. The resources can be logically grouped as projects and API requests. These resources within a project should carry the projectuid in the context. The project scope can be specified in the API request as an HTTP header with the key as `projectuid` and value as the &lt;project uid&gt; The `projectuid` needs to be specified for a request to be applied under the project scope.
+
+**Example**:
+
+While creating a cloud account under a specific project, the request should have the `projectuid.` If the request is submitted without the `projectuid` by default, it will be considered a Tenant Scope request.
```
Comment (prathabk) [inline content/api/1-introduction.md:100]: the request should have the `ProjectUid` in the  http header.

File: content/api/1-introduction.md [hunk 2]
```diff
@@ -93,7 +93,12 @@ The version information is part of the API URI like `v1alpha1`, `v1`. Future API
 
 # Scope
 
-Palette applications operate in *Tenant* and *Project* scopes. The resources can be logically grouped as projects and API requests within a project should carry the project uid in the context. The project scope can be specified in the API request as an HTTP header with the key as `projectUid` and value as the &lt;project uid&gt;
+Palette supports applications to operate under [**Tenant** and **Project**](/user-management/palette-rbac#accessmodes) scope. The resources can be logically grouped as projects and API requests. These resources within a project should carry the projectuid in the context. The project scope can be specified in the API request as an HTTP header with the key as `projectuid` and value as the &lt;project uid&gt; The `projectuid` needs to be specified for a request to be applied under the project scope.
```
[the remark this answers is not in the document]
Comment (resmaa) [reply content/api/1-introduction.md]: changed in all occurrences.

**Review by prathabk — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/902#pullrequestreview-1135240554" title="Approved">prathabk</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/902#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**